.gitignore

```diff
@@ -1,5 +1,6 @@
 # Python
 __pycache__
+.cache
 
 # Vitual Environment
 .venv/
```

Comment:
I think we can add `.pyc` as well. I noticed one of the files committed is `.pyc` file.

Reply:
Done.